src/modules/gendoc.rb

```diff
@@ -10,12 +10,15 @@ def markdown(s)
     lines = s.split("\n")
     newlines = []
     # Backquote function, macro and type names, except if already backquoted and
-    # in code blocks indented by 4 spaces.
+    # link URLs, but not in code blocks indented by 4 spaces.
     lines.each{|l|
         if not l.start_with?('    ')
-            l = l.gsub(/(?<!`)RM_[A-z()]+/){|x| "`#{x}`"}
+            l = l.gsub(/(?<!`)RM_([A-z()]+)/, '`RedisModule_\1`')
             l = l.gsub(/(?<!`)RedisModule[A-z()]+/){|x| "`#{x}`"}
             l = l.gsub(/(?<!`)REDISMODULE_[A-z]+/){|x| "`#{x}`"}
+            l = l.gsub(/(?<![A-z])[a-z_]+\(\)/, '`\0`')
```

Comment:
i think it would be nice to add a line of comment above each of these substitutions to explain what it does.
like: "add backticks around C functions, like malloc()" and such.

Reply:
Sure, will do.